File: q0453/q0453.cpp
class Solution
{
public:
    int minMoves(vector<int> &nums)
    {
        int minNum = INT_MAX;
        for (const int &num : nums)
        {
            minNum = min(minNum, num);
        }

        int moveCount = 0;
        for (const int &num : nums)
        {
            moveCount += num - minNum;
        }
        return moveCount;
    }
};

File: q0453/q0453.java
class Solution {
    public int minMoves(int[] nums) {
        int minNum = Integer.MAX_VALUE;
        for (int num : nums) {
            minNum = Math.min(minNum, num);
        }

        int moveCount = 0;
        for (int num : nums) {
            moveCount += num - minNum;
        }

        return moveCount;
    }
}